config/webpack-config.js

```diff
@@ -19,17 +19,26 @@ var modernConf = {
   // so it needs to be defined here as an external script to ignore for
   // unmet dependency references.
   externals: { jquery: 'jQuery' },
-  context:   path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
-  entry:     scriptsManifest.getDirectoryMap( paths.unprocessed +
-                                              JS_ROUTES_PATH ),
+  cache: true,
+  context: path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
+  entry: scriptsManifest.getDirectoryMap( paths.unprocessed + JS_ROUTES_PATH ),
+  module: {
+    rules: [ {
+      test: /\.js$/,
+      use: [ {
+        loader: 'babel-loader?cacheDirectory=true'
+      } ],
+      exclude: /node_modules/
+    } ]
+  },
   output: {
-    path:     path.join( __dirname, 'js' ),
+    path: path.join( __dirname, 'js' ),
     filename: '[name]'
   },
   plugins: [
-    new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME ),
-    new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME,
-                                             [ COMMON_BUNDLE_NAME ] ),
```

Comment:
You are correct but in order for that to work you need to set the `minChunks` property. The default is infinity. You can test this by setting `minChunks: 3`  and running `gulp build`. It should increase the size of  `common.js` and decrease the size of each of individual modules. That decrease is due to the `standardType` being added to `common.js` as it's required by atleast 3 modules. Do you think we should set the minChunks property?